packages/oid4vci-demo-frontend/src/configs/notary.json

```diff
@@ -160,16 +161,24 @@
           "type": "date",
           "required": true
         }]
-      ]
+      ],
+      "mobile": {
+        "logo": {
+          "src": "notary/logo.svg",
+          "alt": "notary",
+          "width": 150,
+          "height": 150
+        }
+      }
     },
     "SSILandingPage": {
       "photoLeft": "triall/landing_manual_grey.svg",
```

Comment:
Please copy the triall SVG to notary. I don't like that we have all these cross-links to other configs. If we change it there, it will change for other environments as well without us knowing

Reply:
copied all the used asset to the notary folder and changed the reference